internal/pkg/template/templates/overrides/cdk/README.md

```diff
@@ -0,0 +1,23 @@
+# Welcome to overriding your Copilot generated CloudFormation template with the CDK
+
+This is a CDK project with TypeScript to extend the CloudFormation template that gets 
+deployed with AWS Copilot.
+
+The files of special importance are:
+- `package.json` file holds the version of the CDK Toolkit and Library that Copilot will use to apply the overrides.
+- `stack.ts` file holds the transformations to apply to the CloudFormation template.
+- `bin/override.ts` file holds the entrypoint to the CDK application.
+
+## Troubleshooting
+
+* `copilot [noun] package` preview the transformed template by writing to stdout.
+* `copilot [noun] package --diff` show the difference against the template deployed in your environment.
```

Comment:
```suggestion
* `copilot [noun] package --diff` shows the difference against the template deployed in your environment.
```

Reply:
I stole the wording from the CDK readme hehe https://github.com/aws/aws-cdk/blob/87bd42d41f11667aef673d9087c28cecedb87f66/packages/aws-cdk/lib/init-templates/app/typescript/README.md